### PR TITLE
Spec return_call_ref

### PR DESCRIPTION
**Pull request by rossberg**
...as is already implemented in the reference interpreter.

Also, add a missing null case for call_ref reduction.

@conrad-watt, PTAL, it's fairly straightforward.

### CODE REVIEW

**Comment by rossberg** [comment]
@conrad-watt, friendly ping.

File: interpreter/README.md [hunk 1]
```diff
@@ -254,14 +254,13 @@ op:
...
-  return_call <var>
-  return_call_indirect <func_type>
   call <var>
+  call_ref <var>
   call_indirect <var>? <func_type>
-  call_ref
-  return_call_ref
-  func.bind <func_type>
+  return
+  return_call <var>
```
Comment (conrad-watt) [inline interpreter/README.md:261]: Just to check - is the function-references proposal already rebased onto tail-calls? If so, were these entries missed in the tail-call proposal?

**Review by conrad-watt — APPROVED** [review]
lgtm, sorry for delay.

File: interpreter/README.md (hunk 1 above)
Comment (rossberg) [reply interpreter/README.md:261]: Yes, I rebased on the tail calls repo. That contains the instructions, see lines 258/259(old), I just shuffled the order around here.

**Pull request merged**